e2e/jest.test-setup.ts

```diff
@@ -39,7 +41,15 @@ beforeAll(async () => {
   });
 });
 
-// Runs this afterAll() before test's afterAll().
-afterAll(async () => {
+/**
+ * At the end of each test completion, do:
+ * - Disable network interception.
+ * - Delete broswer cookies.
+ * - Reset global page and browser variables.
+ */
```

Comment:
Do you also need an afterAll? Or would that not let you inspect the browser after a failure?

Reply:
I don't think need `afterAll` at this time. We don't set up local server or DB anything like that.